src/Client.php

```diff
@@ -167,8 +167,6 @@ public function flush(?int $timeout = null): PromiseInterface
      *
      * @param array<string, mixed> $payload The payload that will be converted to an Event
      * @param Scope|null           $scope   Optional scope which enriches the Event
-     *
-     * @return Event|null returns ready to send Event, however depending on options it can be discarded
```

Comment:
Why is this removed and not simply fixed?

Reply:
Nothing to fix, I just though it was kinda useless to explain why this method may return either an object or `null` as the official documentation is pretty clear itself and we don't usually document the return values. Do you think it adds value and therefore I should revert this?

Reply:
It's just for consistency, since all other params are documented.